arch/risc-v/src/common/riscv_pmp.c

```diff
@@ -58,7 +58,7 @@
 #define BLOCK_ALIGN_MASK        (MIN_BLOCK_SIZE - 1)
 
 #define PMP_CFG_BITS_CNT        (8)
-#define PMP_CFG_FLAG_MASK       (0xFF)
+#define PMP_CFG_FLAG_MASK       ((uintptr_t)0xFF)
```

Comment:
I think that change is fine, but maybe we can add `ul` or `l` instead of type cast? Just as an option

Reply:
Unfortunately we can't add these because this file applies for both RV32 and RV64.
One portable way to suffix constants is via the `INTxx_C` macros, but there isn't one for PTR type.